run_cartography.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+echo "Starting up..."
+
+# Boto3 has a bug where passing in environment variables does not work as documented in
+# https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html#environment-variables
+echo "Settings AWS Credentials File"
```

Comment:
As in a previous comment, what about `.aws/config`?

Reply:
This is used by boto3 which is aws sdk that cartography uses under the hood. This file is used by that SDK

Reply:
Yea, I don't want to do this but when I only pass in the environment variables, I get the following warning.

`cartography    | WARNING:cartography.intel.aws:No valid AWS credentials could be found. No AWS accounts can be synced. Exiting AWS sync stage.`

It is really quite strange. My goal is to delete this if I can figure out what is going on here. I know the env vars are being passed properly because I can echo them from this shell script. I am trying to debug why boto3 isn't picking up aws env vars